docs/usage/development/updating.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Updating
+---
+import Alert from '@theme/Alert'
+
+<Alert title="Please note" type="warning" icon="alert">
+    Especially during the alpha and beta phases, you are expected to keep all tauri dependencies and toolchains up to date. There is no support for any versions other than latest.
+</Alert>
+
+## Current Latest Versions
+| Component | Description  | Version |
+|-----------|---------|------|
+| [**tauri.js**](https://github.com/tauri-apps/tauri/tree/dev/cli/tauri.js) | CLI and imports | [![](https://img.shields.io/npm/v/tauri.svg)](https://www.npmjs.com/package/tauri) |
+| [**tauri core**](https://github.com/tauri-apps/tauri/tree/dev/tauri) | rust API and tooling | [![](https://img.shields.io/crates/v/tauri.svg)](https://crates.io/crates/tauri)|
+| [**tauri bundler**](https://github.com/tauri-apps/tauri/tree/dev/cli/tauri-bundler) | binary bundler | [![](https://img.shields.io/crates/v/tauri-bundler.svg)](https://crates.io/crates/tauri-bundler)  |
+
+
+## Update NPM Packages
+
+If you are using the `tauri` package:
+```bash
+$ yarn upgrade tauri --latest
+$ npm install tauri@latest
+```
+
```

Comment:
If we end up adding how to check for outdated on the cargo section, it may be worth doing the same for yarn/npm with their `outdated` subcommands.  These by default have the same settings as when using the `-R` flag on `cargo-outdated`.  (and will also show other direct dependencies that are outdated)

```sh
$ npm outdated
Package  Current  Wanted  Latest  Location
tauri      0.9.1   0.8.4   0.9.1  multi-runelite
```

```sh
$ yarn outdated
yarn outdated v1.22.4
info Color legend :
 "<red>"    : Major Update backward-incompatible updates
 "<yellow>" : Minor Update backward-compatible features
 "<green>"  : Patch Update backward-compatible bug fixes
Package Current Wanted Latest Package Type URL                                
tauri   0.8.4   0.8.4  0.9.1  dependencies https://github.com/tauri-apps/tauri#readme
Done in 0.81s.
```

Reply:
like the 2nd comment I added on the cargo one, you can also change the commands to only check for outdated `tauri` package.
`yarn outdated tauri`
`npm outdated tauri`